src/main/java/org/frc5687/steamworks/protobot/commands/DriveWith2Joysticks.java

```diff
@@ -34,7 +34,12 @@ protected void initialize() {
      * @see edu.wpi.first.wpilibj.command.Command#execute()
      */
     protected void execute() {
-        driveTrain.tankDrive(oi.getLeftSpeed(), oi.getRightSpeed());
+        if (oi.getLeftTrigger()){
+            driveTrain.tankDrive(oi.getLeftSpeed(), oi.getRightSpeed());
+        }
+        else {
+            driveTrain.oneStickDrive(oi.getRightSpeed(), false);
```

Comment:
Would it be easier here to just call driveTrain.tankDrive(oi.getRightSpeed(), oi.getRightSpeed(), false) than making an all new method?